kinto_admin/__init__.py

```diff
@@ -1,11 +1,16 @@
+import pkg_resources
 from pyramid.static import static_view
 
+#: Module version, as defined in PEP-0396.
+__version__ = pkg_resources.get_distribution("kinto-dist").version
+
 
 def includeme(config):
     # Process settings to remove storage wording.
 
     # Expose capability.
     config.add_api_capability("admin",
+                              version=__version__,
```

Comment:
Maybe this is exactly what we would expect here. Shouldn't we put the kinto-admin version here instead ?

Should we set kinto-dist version in the `kinto.project_version` entry ?